docs/www/configuration.md

```diff
@@ -622,6 +622,19 @@ rpk:
   # Default: ''
   coredump_dir: "/var/lib/redpanda/coredump"
 
+  # Creates a "ballast" file that can be removed in the odd case that a redpanda node
+  # runs out of space, so that the node can be restarted and troubleshot.
+  # Default: false
+  tune_ballast_file: false
+
+  # The path where the ballast file will be created.
+  # Default: "/var/lib/redpanda/data/ballast.txt"
+  ballast_file_path: "/var/lib/redpanda/data/ballast.txt"
```

Comment:
I don't think we should use a `.txt` suffix for a binary file that is filled with zeroes. I think we can just leave out a suffix.

Reply:
Good point. No one should try to open it haha.

Reply:
Done.

Reply:
Done.